addons/website/data/website_contactus_data.xml

```diff
@@ -0,0 +1,168 @@
+                                </section>
+                            </div>
+                            <div class="col-lg-4 mt-4 mt-lg-0">
+                                <p class="p-0 m-0">My Company</p>
+                                <ul class="list-unstyled mb-0 pl-2">
+                                    <li><i class="fa fa-map-marker fa-fw mr-2" /><span class="o_force_ltr">Chaussée de Namur 40</span></li>
+                                    <li><i class="fa fa-phone fa-fw mr-2" /><span class="o_force_ltr">+ 32 81 81 37 00</span></li>
+                                    <li><i class="fa fa-1x fa-fw fa-envelope mr-2" /><span>info@yourcompany.example.com</span></li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                </section>
+            </div>
+        </t>
+    </template>
+
+    <record id="website.contactus_thanks" model="website.page">
+        <field name="name">Thanks (Contact us)</field>
+        <field name="type">qweb</field>
+        <field name="url">/contactus-thank-you</field>
+        <field name="website_indexed" eval="False"/>
+        <field name="is_published">True</field>
+        <field name="key">website.contactus_thanks</field>
+        <field name="arch" type="xml">
+            <t name="Thanks (Contact us)" t-name="website.contactus_thanks">
+                <t t-call="website.layout">
+                    <div id="wrap" class="oe_structure">
+                        <section class="s_text_block pt40 pb40 o_colored_level " data-snippet="s_text_block">
+                            <div class="s_allow_columns container">
+                                <div class="row">
+                                    <div class="col-lg-7 col-xl-6 mr-lg-auto">
+                                        <span class="d-block fa fa-4x fa-thumbs-up mx-auto rounded-circle bg-primary" />
+                                        <br />
+                                        <h1 class="text-center">Thank You!</h1>
+                                        <div class="pb16 pt16 s_hr" data-snippet="s_hr" data-name="Separator">
+                                            <hr class="mx-auto border-top w-50 border-dark text-center" />
+                                        </div>
+                                        <h5 class="text-center">
+                                            <span class="fa fa-check-circle" />
+                                            <span>Your message has been sent <b>successfully</b></span>
+                                        </h5>
+                                        <p class="text-center">We will get back to you shortly.</p>
+                                    </div>
+                                    <div class="col-lg-4">
+                                        <p class="p-0 m-0">My Company</p>
+                                        <ul class="list-unstyled mb-0 pl-2">
```

Comment:
```suggestion
                                        <ul class="list-unstyled mb-0 pl-2">
                                             <li>My Company</li>
```
Diff is weird but I'd just move the My Company from the `<p>` to a new `<li>`, so:
1. We don't need those "I use a `<p>` but I don't want the `<p>` style" classes (p-0 m-0)
2. It will keep the same layout/design as before without an extra indent visually

See 
![image](https://user-images.githubusercontent.com/30048408/118803455-a9083700-b8a3-11eb-997c-f2b3957e3668.png)
First "My Company" is in a `<p>` while second is in `<li>`